controllers/perconaservermysqlbackup_controller.go

```diff
@@ -230,6 +230,10 @@ func (r *PerconaServerMySQLBackupReconciler) Reconcile(ctx context.Context, req
 			return rr, errors.Errorf("storage type %s is not supported", storage.Type)
 		}
 
+		status.Image = cluster.Spec.Backup.Image
+		status.SSLSecretName = cluster.Spec.SSLSecretName
```

Comment:
what's the point of having this in status?

Reply:
@egegunes It was added in https://github.com/percona/percona-server-mysql-operator/pull/158. We need this for `delete-backup` finalizer job when cluster doesn't exist